src/Integration.Vsix.UnitTests/packages.lock.json

```diff
@@ -413,14 +413,6 @@
         "resolved": "7.10.6070",
         "contentHash": "p7xNzxIoZWDqSGlCi84u1CsNSL9YL7HWBdurfBUW8TZOsfW/0DS8/2MQcicQZav2Csq1f7Pep+VRdZJZkmQKJA=="
       },
-      "Microsoft.VisualStudio.SDK.Analyzers": {
```

Comment:
This project never needed the VSSDK analyzers. I think they used to appear here via a transitive reference from the Integration.Vsix project.
However, all of the VSSDK build tools assets are now marked as `private` in the Integration.VSIX project so they don't flow downstream.